app/utils/Utils.scala

```diff
@@ -23,6 +23,8 @@ import scala.util.Try
 object Utils {
   val acceptHeaderPattern = "^application/vnd[.]{1}hmrc[.]{1}(.*?)[+]{1}(.*)$".r
 
+  val acceptHeaderMissing = "The accept header is missing or invalid"
```

Comment:
I'm not a fan of this living here, if we're going to be having stock messaging, we should probably isolate that. I'm also not sure if we have to support welsh here as its an API - I assume not, but at the least we can move this to a Messages object instead of Utils.